src/script/ptl-tool.py

```diff
@@ -293,16 +295,14 @@ def main():
     else:
         argv = sys.argv[1:]
     parser.add_argument('--branch', dest='branch', action='store', default=default_branch, help='branch to create ("HEAD" leaves HEAD detached; i.e. no branch is made)')
-    parser.add_argument('--merge-branch-name', dest='merge_branch_name', action='store', help='name of the branch for merge messages')
+    parser.add_argument('--merge-branch-name', dest='merge_branch_name', action='store', default=False, help='name of the branch for merge messages')
```

Comment:
@batrick I don't understand what is the relevance of `--merge-branch-name` now. I think this option be removed, as branch name is configurable and automated by appending time to it. Am I missing something here ?  Please clarify. Thanks.

Reply:
It is useful in conjunction with `--branch HEAD`. I use it frequently when merging PRs to master:

`ptl-tool.py --branch HEAD --merge-branch-name master 12345`

Reply:
(A peculiarity of my setup is that I don' thave a local master branch because I find it useless.)